dolphinscheduler-api/src/main/resources/application.yaml

```
@@ -113,9 +113,9 @@ registry:
       base-sleep-time: 60ms
       max-sleep: 300ms
       max-retries: 5
-    session-timeout: 30s
-    connection-timeout: 9s
-    block-until-connected: 600ms
+    session-timeout: 35s
+    connection-timeout: 35s
+    block-until-connected: 30s
```

It's better to use the curator default setting rather than give a untest value.
```suggestion
    session-timeout: 60s
    connection-timeout: 15s
    block-until-connected: 15s
```

---

Okay, I'll adjust it. Thank you

---

> 最好使用 curator 默认设置，而不是给出未测试的值。最好使用策展人默认设置，而不是给出未测试的值。

It has been adjusted, please review, thank you